src/google/protobuf/compiler/objectivec/objectivec_helpers.cc

```diff
@@ -144,6 +150,38 @@ PrefixModeStorage::PrefixModeStorage() {
   }
 }
 
+std::string PrefixModeStorage::objc_class_prefix(const FileDescriptor* file) {
```

Comment:
should this take a reference so it don't have to worry about getting a nullptr?

Reply:
I can update it, this was mostly copy/pasta from other invocations.